.azure/azure-linux-template.yml

```diff
@@ -51,7 +63,16 @@ jobs:
           hlint test/ --cpp-simple
           ;;
         stack)
-          stack test --haddock --no-haddock-deps --ghc-options="$GHC_OPTIONS"
+          stack $ARGS test --haddock --no-haddock-deps --ghc-options="$GHC_OPTIONS" --copy-bins --local-bin-path bin
+
+          # Get output about whether the exe is dynamically linked
+          # Ignore the exit code though, we don't want to fail just because it is static
+          ldd ./bin/stack || true
```

Comment:
Since this always succeeds, is it intended to remain here or was it just intended for debugging?  Should we `grep` the output for an expected string (depending on whether we want static or dynamic) and fail if it's wrong?

Reply:
Originally I just meant it as debug output, but now I think we should add a check. No `grep` is necessary, we can pay attention to the exit code of `ldd` I think.